snaps_tests/__init__.py

```diff
@@ -93,7 +93,7 @@ def _get_latest_ssh_private_key():
 
 class SnapsTestCase(testtools.TestCase):
 
-    snap_content_dir = None
+    snap_content_dir = ''
```

Comment:
Why are you changing this here? Isn't it `None` so that tests which don't set it fail?

Reply:
Ah. Actually removing it is even better. Nice.